README.md

```diff
@@ -362,6 +391,22 @@ Both of these strategies fall over when considering that multiple pieces of code
 
 ###### Conclusion: No feasible strategy for deadlock avoidance
 
+#### Doesn't this proposal lack some kind of symmetry, in terms of `import`/`export` or `async`/`await`?
+
+If there's `await`, the logical thing would be for a corresponding `async`. Or, if you decorate the `import` with `await`, you may expect that there be something corresponding on the `export`.
+
+Unfortunately, neither of these make sense here. The logical unit which is asynchronous here is the module itself. However, there's no syntactic signifier in JavaScript to indicate the module, so noplace to hang the `async` off of. And, it would be strange to declare each export `async` as well--modules with no exports can contain a top-level `await` as well, and multiple `async`s in a file would be redundant.
+
+Therefore, this proposal relies on whether the module contains a top-level `await` to determine whether it's synchronous or asynchronous.
+
+#### Wouldn't `import await` be viral?
+
+Just like `async` functions, modules which contain top-level `await` need to be `await`-ed to use their result. And also like `async` functions, modules containing top-level `await` can be invoked to get their result as a `Promise`, which can be passed around flexibly before `await`-ing it. (In the case of modules, that call is through `import()`.)
+
+`async` functions and top-level `await` can be thought of as being "viral" because they have to be considered all the way up the call stack/dependency chain. You can't just change an ordinary function to an async function without considering its users, who would probably be broken, unless they happen to always `await` the result. It would be quite unusual for them to do such defensive `await`-ing, and we don't seem to see that catching on as an idiom.
+
+The same applies to top-level `await`--their dependencies need to `await` the module when importing, and recursively so. This means that, just like changing a function into an `async` function, it's a semver-major change to add top-level `await` to a module. We will need to document this effect carefully, to avoid potential ecosystem breakage.
+
 ## History
```

Comment:
#### What's the difference between `import await "module"` and `await import("module")`?

`import await` starts fetching during the `Fetch and Parse` phase, while `await import()` starts fetching during the `Evaluation` phase.

Multiple `import await` will run in parallel, while multiple `await import()` will run sequential.

Exports of `import await` can be analysed statically and incorrect export names will result in a SyntaxError. Analyzing `await import()` is harder and incorrect export names will result in `undefined` and a RuntimeError.

`import await` is hoisted, `await import()` is an Expression and not hoisted.

`import await` can only be placed on top-level, while `await import()` can be placed at top-level or in async functions at any location where an Expression is allowed. 

`import await` must be used with a String Literal, while `await import()` can be used with any expression as argument (``await import(`./data/${value}.js`)``)

Reply:
Thanks for writing this! I added it to the FAQ.